src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEmbeddedAttributeSymbol.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Cci;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedEmbeddedAttributeSymbol : NamedTypeSymbol
```

Comment:
>SynthesizedEmbeddedAttributeSymbol [](start = 26, length = 34)

Is code in this file temporary solution? Please add a PROTOTYPE comment.

Reply:
The class is based on the implementation from the `features/readonly-ref` branch, and should be merged with that implementation.

Reply:
> and should be merged with that implementation.

Merged or replaced? What isn't handled by the code in features/readonly-ref branch?

Reply:
The `features/readonly-ref` implementation assumes a single parameter-less constructor currently.

Reply:
I think we should clearly indicate (if possible) what portions of code should be replaced and what should be merged.

Reply:
Added comment.